core.fnl

```diff
@@ -115,8 +116,10 @@ Returns nil. This function causes side-effects.
   Returns true if file extension ends in .fnl or .lua
   "
   (let [ext (split "%p" file)]
-    (or (contains? "fnl" ext)
-        (contains? "lua" ext))))
+    (and
+     (or (contains? "fnl" ext)
+         (contains? "lua" ext))
+     (not (string.match file "-test")))))
```

Comment:
This could, but is unlikely to bite you in the future, since it doesn't care where this shows up in the string.

Reply:
Fair point